server/vulnerabilities/db.go

```diff
@@ -21,37 +21,56 @@ func sqliteDB(dbPath string) (*sqlx.DB, error) {
 }
 
 func applyCPEDatabaseSchema(db *sqlx.DB) error {
+	// Use a new table cpe_2 containing new columns vendor, product. view cpe used for backwards compatibility
+	// with old fleet versions that use "select * from cpe ...". When creating the view, we need to
+	// select rowid because it is used for joins between the cpe and cpe_search tables
 	_, err := db.Exec(`
-	CREATE TABLE IF NOT EXISTS cpe (
-		cpe23 TEXT NOT NULL,
-		title TEXT NOT NULL,
-		version TEXT,
-		target_sw TEXT,
-		deprecated BOOLEAN DEFAULT FALSE
-	);
-	CREATE TABLE IF NOT EXISTS deprecated_by (
-		cpe_id INTEGER,
-		cpe23 TEXT NOT NULL,
-		FOREIGN KEY(cpe_id) REFERENCES cpe(rowid)
-	);
-	CREATE VIRTUAL TABLE IF NOT EXISTS cpe_search USING fts5(title, target_sw);
-	CREATE INDEX IF NOT EXISTS idx_version ON cpe (version);
-	CREATE INDEX IF NOT EXISTS idx_cpe23 ON cpe (cpe23);
-	CREATE INDEX IF NOT EXISTS idx_target_sw ON cpe (target_sw);
-	CREATE INDEX IF NOT EXISTS idx_deprecated_by ON deprecated_by (cpe23);
-	`)
+CREATE TABLE IF NOT EXISTS cpe_2 (
```

Comment:
Makes me wonder: Do we really need the `IF NOT EXISTS` in all of these? (`GenerateCPEDB` is only called by [nvd's generate workflow](https://github.com/fleetdm/nvd/blob/master/.github/workflows/generate.yml))